code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -212,6 +212,9 @@
 				if (prob(stunprob) && M.powerlevel >= 8)
 					adjustFireLoss(M.powerlevel * rand(6,10))
 					updatehealth()
+				if (M.transformeffects & SLIME_EFFECT_ORANGE) // TEGU EDIT
```

Comment:
You should put '// Tegustation Xenobio Crossbreeds edit' instead, and add it to the README.md in the ModularTegustation folder

Reply:
I mainly use those to make /tg/ merges easier, but yeah, you have a point.